products/sonarqube.md

```diff
@@ -21,8 +21,8 @@ releases:
     releaseDate: 2021-07-02
     support: true
     eol: false
-    latest: "9.8.0"
-    latestReleaseDate: 2022-12-19
+    latest: "9.9.0"
```

Comment:
9.9.0 is not yet announced (but the docker image has been published on https://hub.docker.com/_/sonarqube/).